src/psyclone/psyir/backend/fortran.py

```diff
@@ -41,8 +41,8 @@
 '''
 
 from psyclone.psyir.frontend.fparser2 import Fparser2Reader
+from psyclone.psyir.symbols import DataSymbol, ArgumentInterface
```

Comment:
pylint complains about the order of these imports.

Reply:
fixed